File: src/Contacts/index/get.py
  import requests
  import json

  url = "https://sms.velstack.com/api/contact"

  headers = {
        "Content-Type": "application/json",
        "Authorization": "Bearer API_KEY"
        }

  response = requests.get(url, headers=headers)

  print(response.json())

File: src/Contacts/index/get.js
import axios from 'axios';



axios.get('https://sms.velstack.com/api/contact', {
    headers: {
        Authorization: "Bearer " + "API_KEY",
        Accept: "application/json",
    }
}).then((response) => {
   const data = response.data

}).catch((errors) => {
    alert(errors.response.data.message)
})